ui/tailwind.config.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
         'medium-blue': '#1378FF',
         'light-blue': '#37ADEB',
         grey: '#979797',
-        'corsha-blue-light': '#10445799',
+        'corsha-blue-light': 'rgb(22,49,60)',
```

Comment:
We could probably keep this to as hex to match the others. Where's the source of truth on these values?
```suggestion
        'corsha-blue-light': '#16313C',
```

Reply:
RGB and the hex code actually do appear to have differences in practice (at least in my testing environment, Firefox on Mac). Settling on RGB(22,49,60) was an _incredibly_ painful process trying to match a color from a PowerPoint template in code. The source of truth for this value is "This specific value, phrased in RGB, causes Mac's Digital Color Meter app to report the same color between the PowerPoint and the web page".

Things that did not work, listed here for completeness:
- Our actual brand color, which displays differently for some incredibly absurd reason I don't understand
- The same color specified as hex
- The actual sRGB value coming from Color Meter when waved over the powerpoint. No, really, try it. You write down the color that Color Meter says it is, and a _slightly different color_ gets rendered. It was driving me mad

Reply:
Looked more into it, is this used anywhere other than the design system page right now? In that case we could probably come back to this since we've discussed that changes may need to be made there as well.